backend/app/services/codegraph/repo_service.py

```
@@ -0,0 +1,285 @@
+                        "Authorization": self.secret_token
+                    }
+                )
+
+            await self.supabase.table("indexed_repositories").update({
+                "is_deleted": True,
+                "updated_at": datetime.now().isoformat()
+            }).eq("repository_full_name", repo_full_name).eq("is_deleted", False).execute()
+
+            return {"status": "success", "repo": repo_full_name, "graph_name": graph_name}
+
+        except Exception as e:
```

Ruff (0.13.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)